### PR TITLE
Clean up new RPC-IRR instances

### PR DESCRIPTION
**Pull request by mattt416**
This commit updates periodic_cleanup.py so we can pass a pattern to use
when cleaning up instances.  We also update the periodic cleanup job to
cleanup ra|ri instances.

Connects https://github.com/rcbops/u-suk-dev/issues/1655

### CODE REVIEW

**Comment by mattt416** [comment]
Not tested yet -- please hold off on merging until this has been reviewed and tested.  :)

**Comment by mattt416** [comment]
Tested here: https://rpc.jenkins.cit.rackspace.net/job/Periodic-Cleanup/555/console

Didn't clean up anything as far as I can tell, but didn't blow anyway anything either.  :)

**Comment by mkam** [comment]
Can you update jenkins_node.py too now that it matches prefixes there as well?

**Comment by mkam** [comment]
Rebased and added the regex matching to the jenkins node deletion as well:

https://rpc.jenkins.cit.rackspace.net/job/Periodic-Cleanup/585/console

File: scripts/periodic_cleanup.py [hunk 1]
```diff
@@ -34,7 +35,7 @@ def cleanup_instances(cs_client, age_limit, instance_prefix):
```
Comment (git-harry) [inline scripts/periodic_cleanup.py]: It's probably worth removing the default of `"ra"` on line 21.

Comment (mkam) [inline scripts/periodic_cleanup.py]: Done!

**Pull request merged**